UM/CentralFileStorage.py

```diff
@@ -34,59 +35,62 @@ class CentralFileStorage:
     """
 
     @classmethod
-    def store(cls, file_path: str, file_id: str, version: Version = Version("1.0.0")) -> None:
+    def store(cls, path: str, path_id: str, version: Version = Version("1.0.0")) -> None:
         """
         Store a new file into the central file storage. This file will get moved to a storage location that is not
         specific to this version of the application.
 
         If the file already exists, this will check if it's the same file. If the file is not the same, it raises a
         `FileExistsError`. If the file is the same, no error is raised and the file to store is simply deleted. It is a
         duplicate of the file already stored.
-        :param file_path: The path to the file to store in the central file storage.
-        :param file_id: A name for the file to store.
+        :param path: The path to the file to store in the central file storage.
+        :param path_id: A name for the file to store.
         :param version: A version number for the file.
         :raises FileExistsError: There is already a centrally stored file with that name and version, but it's
         different.
         """
         if not os.path.exists(cls._centralStorageLocation()):
             os.makedirs(cls._centralStorageLocation())
-        if not os.path.exists(file_path):
-            Logger.debug(f"{file_id} {str(version)} was already stored centrally or the provided file_path is not correct")
+        if not os.path.exists(path):
+            Logger.debug(f"{path_id} {str(version)} was already stored centrally or the provided file_path is not correct")
             return
 
-        storage_path = cls._getFilePath(file_id, version)
+        storage_path = cls._getFilePath(path_id, version)
 
         if os.path.exists(storage_path):  # File already exists. Check if it's the same.
-            if os.path.getsize(file_path) != os.path.getsize(storage_path):  # As quick check if files are the same, check their file sizes.
-                raise FileExistsError(f"Central file storage already has a file with ID {file_id} and version {str(version)}, but it's different.")
-            new_file_hash = cls._hashFile(file_path)
-            stored_file_hash = cls._hashFile(storage_path)
+            if os.path.getsize(path) != os.path.getsize(storage_path):  # As quick check if files are the same, check their file sizes.
+                raise FileExistsError(f"Central file storage already has a file with ID {path_id} and version {str(version)}, but it's different.")
+            new_file_hash = cls._hashPath(path)
```

Comment:
Perhaps rename this to "new_item_hash" as well (since it might not be a file but a dir?)